bpf/bpf_host.c

```diff
@@ -602,6 +602,7 @@ handle_ipv4(struct __ctx_buff *ctx, __u32 secctx __maybe_unused,
 #endif /* ENABLE_HOST_FIREWALL */
 	void *data, *data_end;
 	struct iphdr *ip4;
+	__s64 fraginfo __maybe_unused;
```

Comment:
Nit: I'd prefer typedef'ing `fraginfo_t` instead of using plain `__s64`. So that people can reach to this nice description by single tag jump and know what it is.

https://github.com/cilium/cilium/pull/38202/commits/dc82a4e712aaa428f83dc647bc30249da6adeacb#diff-6942a0e4e719b2ff83b78100ab11f7cd2a9daf65fb3b145780ae3910442c2e4cR9-R31